src/game/camera/camerastyle.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2020 The reone project contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+
+namespace reone {
+
+namespace resource {
+    class TwoDaRow;
+}
+
+namespace game {
+
+struct CameraStyle {
+    float distance{ 0.0f };
```

Comment:
Please add a whitespace between each member name and its initializer (for uniformity).